update/files/multiFileWriter.go

```diff
@@ -53,8 +59,9 @@ func (m *multiFileWriter) NewFile(fileName string) error {
 		return nil
 	}
 
-	file, err := os.OpenFile(m.exportFolder+"/"+fileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	file, err := os.OpenFile(m.exportFolder+"/"+fileName, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
 	if err != nil {
+		_ = os.Remove(m.exportFolder + "/" + fileName)
 		log.Debug("unable to open file")
```

Comment:
might log additional info here as the name of the file

Reply:
done.